runtime/doc/lsp.txt

```diff
@@ -1041,7 +1041,9 @@ enable({name}, {enable})                                    *vim.lsp.enable()*
     Parameters: ~
       • {name}    (`string|string[]`) Name(s) of client(s) to enable.
       • {enable}  (`boolean?`) `true|nil` to enable, `false` to disable
-                  (actively stops and detaches clients as needed)
+                  (actively stops and detaches clients as needed, and force
+                  stops them if necessary after `client.flags.exit_timeout`
+                  milliseconds, with a default time of 200 milliseconds)
```

Comment:
Given that force terminating a server can lead to project settings/index files getting corrupted I think this shouldn't have any timeout by default